### DISCUSSION TITLE
What is the behavior when more than one sonde is set and received?

### DISCUSSION THREAD

**Discussion by eben80** [dl9rdz/rdz_ttgo_sonde#385] [Q&A] [answered]
If more than one sonde is set enabled and received, what is the software behavior? Does it keep receiving the first one it finds or does it alternate between the enabled frequencies even if not in range?

**Comment by hr-ru** [comment, marked as answer]
The default behaviour is that the software stays on one frequency as long as a signal is received on that frequency.

Some people have modified the screens file to enable some alternation: You could set the RX timer of some screen to some value > 0, causing the software to (probably the better choice) go back to the Scanner (action O) or go to the next frequency (action +)

(And there is a plan to move this to the actual config to make it easier to configure)

**Reply by eben80** [reply]
So my current Scanner config is:
@Scanner
timer=-1,0,0
key1action=D,#,F,W
key2action=#,#,#,#
timeaction=#,D,+
0,0=XScan
0,5=S#:
0,9=T
3,0=F MHz
5,0=S
7,5=n

Are the changes I need to do in the timeaction section?

**Reply by dl9rdz** [reply]
No, not in the scanner config.

What you want is: If you are receiving a sonde - and that means a different view is used, by default the Legacy (for LCD) or MainTFT (for TFT) view - you want to rotate between different frequencies. In that config you need to change the timer and timeaction.
For example, for TFT, change
```
@MainTFT
timer=-1,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,0
```
into
```
@MainTFT
timer=-1,30,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,0,0
```

In order to go back to the scanner (and let the scanner move on to the next frequency) after continuously receiving something for 30 seconds. In this case a single missing frame will restart the timer. If you don't want this, you can also switch after 30 seconds on a frequency unconditionally, using the first timer:
```
@MainTFT
timer=30,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=0,#,0
```

Note that this was not designed with this use case in mind, and I have not used this myself, but I was told by several people that they are using this...

**Reply by eben80** [reply]
Thank you for the thorough response, I will give it a go on my stationary device.

**Comment by OneThreeCharlie** [comment]
@eben80 - I've been looking for this feature, I'm in a geographical position where i can track more than one sonde at a time. Is there a BIN I can flash my TTGO with that has the above changes made?
